blocks/video-promo-block/features/video-promo/default.jsx

```diff
@@ -18,36 +20,58 @@ const DescriptionText = styled.p`
 `;
 
 const AlertBadge = styled.span`
+  background-color: #db0a07;
+  border-radius: 5px;
+  color: #fff;
   display: inline-block;
   padding: 0.3rem 0.8rem;
-  color: #FFFFFF;
-  background-color: #DB0A07;
-  border-radius: 5px;
 `;
 
 const VideoPromo = ({ customFields }) => {
   const { arcSite } = useFusionContext();
   const {
-    uuid,
     autoplay = false,
```

Comment:
autoplay is already default false based on proptypes 

![Screen Shot 2020-11-19 at 14 24 34](https://user-images.githubusercontent.com/5950956/99720137-ea459880-2a72-11eb-9373-4c90567dd650.png)
```suggestion
    autoplay,
```